dashboard/app/templates.html

```diff
@@ -283,3 +283,44 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 	{{end}}
 {{end}}
 {{end}}
+
+{{/* Show crashes */}}
+{{define "crashes"}}
```

Comment:
2 other table templates in this file are called bug_list and manager_list, please call this consistently with the existing code: crash_list.